src/armscan_env/envs/state_action.py

```diff
@@ -43,7 +47,7 @@ def to_normalized_array(
         result = np.concatenate([rotation, translation])
 
         if not (result >= -1).all() or not (result <= 1).all():
-            raise ValueError(
```

Comment:
why this change?

Reply:
because this might happen, and it is dealt with it. Per see, the action is still valid, it is just not possible to get such an action from the agent